bin/mtt.ml

```diff
@@ -4,6 +4,19 @@ open Result.Let_syntax
 open Mtt
 open ParserInterface
 
+let formatLError Location.{ data = error; loc } =
```

Comment:
Sorry, I missed the following two identifiers, let's use more idiomatic names here.
Also, error messages should have location information by default, hence I suggest we change it as follows:
```suggestion
let format_error Location.{ data = error; loc } =
```

Reply:
Ok, thank you. I forgot to check naming too after the first review.